sys/net/routing/rpl/rpl_control_messages.c

```diff
@@ -539,19 +532,6 @@ void rpl_recv_DIO(void)
             return;
         }
     }
-    else if (my_inst == NULL) {
-        DEBUGF("Not joined an instance yet\n");
-    }
```

Comment:
I thought it could be good to keep an overview if this node will join the dodag because it hasn't joined any dodag or because it prefers the dodag its DIO it is currently receiving.